airbyte-ci/connectors/pipelines/pipelines/pipelines/connectors.py

```diff
@@ -86,7 +86,7 @@ async def run_connectors_pipelines(
         # HACK: This is to get a long running dockerd service to be shared across all the connectors pipelines
         # Using the "normal" service binding leads to restart of dockerd during pipeline run that can cause corrupted docker state
         # See https://github.com/airbytehq/airbyte/issues/27233
-        dockerd_service = environments.with_global_dockerd_service(dagger_client)
+        dockerd_service = environments.with_dockerd_service(dagger_client)
```

Comment:
This seems like a good candidate for the hacks file!

Reply:
👍 ed158f42bf6a5879c3ab70243d47aad04534b3e7